### ISSUE TITLE
Pin start, stop degrees

### ISSUE THREAD

**Issue by AccuBytes-TimJ** [joyja/ignition-framework#2]
It would be handy, if possible, to give a parameter for start degrees and stop degrees, so a user could customize it.  For example, if they wanted it to start at 6:00 rather than 3:00, or perhaps only use 300 degrees of the circle with an open gap at the bottom.

**Comment by joyja** [comment]
Good suggestions! Changing the start from 3:00 to 6:00 (or any other point) is trivial while start and stop params to form a gap near the bottom will take some thought, though should be do-able. The gauge uses SVG stroke properties to form the circular bar, so just a matter of playing with the dash setting bindings. I'll take a look or let me know if you've come up with something as well.

**Comment by joyja** [comment]
Updated in master branch (at some point I'll version this thing) - - - circular gauge now has two new params:

- **rotate:** sets the start angle of the gauge. For example 90 deg will cause the guage to start from 6:00
- **gap:** creates a gap at the origin of the gauge, this setting on either side of zero. For example, a setting of 15 deg will create a 30 deg gap with the middle of it set on the gauge origin.

I added them to the demo/docs here to: https://ignition.jarautomation.io/data/perspective/client/playground/circular
